yaksh/live_server_tests/load_test.py

```diff
@@ -30,7 +30,20 @@ def setUpClass(cls):
 
         app_label = 'yaksh'
         group_name = 'moderator'
-        cls.group = create_group(group_name, app_label)
+
+        try:
+            cls.group = Group.objects.get(name=group_name)
+        except Group.DoesNotExist:
+            cls.group = Group(name=group_name)
+            cls.group.save()
```

Comment:
You don't need try and except in this case. instead of `get` use `get_or_create` and remove try except.